rebench/tests/persistency_test.py

```diff
@@ -35,8 +40,9 @@
 from ..rebench import ReBench
 
 
-class PersistencyTest(ReBenchTestCase):
 
```

Comment:
Could you revert the whitespace changes here?
For reviewing, it's generally helpful to avoid unnecesary changes.